backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt

```diff
@@ -157,6 +157,14 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     @Argument(value = "-Xruntime", deprecatedName = "--runtime", valueDescription = "<path>", description = "Override standard 'runtime.bc' location")
     var runtimeFile: String? = null
 
+    @Argument(
+        value = SOURCE_LIBRARY_ARG,
+        valueDescription = "<path>",
+        description = "A library to be processed in the same manner as source files (test processing, ObjC export etc).\n" +
```

Comment:
> (test processing, ObjC export etc)

Is this meaningful for people reading command-line compiler help?

Reply:
Probably not. Will remove it.